user_templates/mcu_defaults/mmu/BTT_MMB_CAN_v1.1.cfg

```diff
@@ -0,0 +1,23 @@
+[board_pins mmu_manufacturer]
+mcu: mmu
+aliases:
+    MCU_M1_STEP=PB15   , MCU_M1_DIR=PB14   , MCU_M1_EN=PB8    , MCU_M1_UART=PA10   , # MCU_M1_DIAG=PA3   ,
+    MCU_M2_STEP=PD2    , MCU_M2_DIR=PB13   , MCU_M2_EN=PD1    , MCU_M2_UART=PC7    , # MCU_M2_DIAG=PA4   ,
+    MCU_M3_STEP=PD0    , MCU_M3_DIR=PD3    , MCU_M3_EN=PA15   , MCU_M3_UART=PC6    , MCU_M3_DIAG=PB9   ,
+    MCU_M4_STEP=PB6    , MCU_M4_DIR=PB7    , MCU_M4_EN=PB5    , MCU_M4_UART=PA9    , # MCU_M4_DIAG=PA8   ,
+
+    MCU_MISO=PA6  , MCU_SCK=PA5  , MCU_MOSI=PA7  ,
+
+    MCU_STP1=PA3 , MCU_STP2=PA4  , MCU_STP3=PB9  , # Shared with Steppers Diag pins  1-2-4 so it's better to use STP 4-11 for others servos/sensors
+    MCU_STP4=PA8 , MCU_STP5=PC15 , MCU_STP6=PC13 , MCU_STP7=PC14 , MCU_STP8=PB12 , MCU_STP9=PB11 , MCU_STP10=PB10 , MCU_STP11=PB2 ,
+
+
+    MCU_MOT=PA0      ,
+    MCU_SENSOR=PA1   ,
+    MCU_RGB=PA2      ,
+
+    ## I2C header
+    MCU_I2C_SDA=PB4  , MCU_I2C_SCL=PB3  ,
+
+    ## SWD header
+    MCU_SWD_PA14=PA14 , MCU_SWD_PA13=PA13 ,
```

Comment:
This file need to be modified to use the pins aliases used in Klippain.

Look at this file for documentation: https://github.com/Frix-x/klippain/blob/main/docs/pinout.md
And as an example, you can look at the MMB v1.0 user template here: https://github.com/Frix-x/klippain/blob/main/user_templates/mcu_defaults/mmu/BTT_MMB_CAN_v1.0.cfg